reason-react-native/src/apis/Style.md

```diff
@@ -4,4 +4,1298 @@ title: Style
+- `` `_500 ``
+- `` `_600 ``
+- `` `_700 ``
+- `` `_800 ``
+- `` `_900 ``
+
+- [Web reference](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight)
+
+#### `includeFontPadding`
+
+Accepts a `bool`.
+
+#### `letterSpacing`
+
+Accepts a `float`.
+
+- [Web reference](https://developer.mozilla.org/en-US/docs/Web/CSS/letter-spacing)
+
+#### `lineHeight`
+
+Accepts a `float`.
+
+Only accepts logical pixels.
+
+- [Web reference](https://developer.mozilla.org/en-US/docs/Web/CSS/letter-spacing)
+
+#### `textAlign`
+
+Accepts one of the following values:
+
+- `` `auto ``
+- `` `left ``
+- `` `right ``
+- `` `center ``
+- `` `justify ``
+
+- [Web reference](https://developer.mozilla.org/en-US/docs/Web/CSS/text-align)
+
+#### `textAlignVertical`
+
+Accepts one of the following values:
+
+- `` `auto ``
+- `` `top ``
+- `` `bottom ``
+- `` `center ``
+
+#### `textDecorationColor`
+
+Accepts a `Color.t` (`string`).
+
+- [Web reference](https://developer.mozilla.org/en-US/docs/Web/CSS/text-decoration-color)
+
+#### `textDecorationLine`
+
+Accepts one of the following values:
+
+- `` `none ``
+- `` `underline ``
+- `` `lineThrough ``
+- `` `underlineLineThrough ``
+
+- [Web reference](https://developer.mozilla.org/en-US/docs/Web/CSS/text-decoration-line)
+
+#### `textDecorationStyle`
+
+Accepts one of the following values:
+
+- `` `solid ``
+- `` `double ``
+- `` `dotted ``
+- `` `dashed ``
+
+- [Web reference](https://developer.mozilla.org/en-US/docs/Web/CSS/text-decoration-style)
+
+#### `textShadowColor`
+
+Accepts a `Color.t` (`string`).
+
+You can see it as CSS `text-shadow` blur radius.
+
+- [Web reference](https://developer.mozilla.org/en-US/docs/Web/CSS/text-shadow)
+
+#### `textShadowOffset`
+
+Accepts an `offset`.
+
+You can see it as CSS `text-shadow` offsets.
+
+- [Web reference](https://developer.mozilla.org/en-US/docs/Web/CSS/text-shadow)
+
+#### `textShadowRadius`
+
+Accepts a `float`.
+
+You can see it as CSS `text-shadow` blur radius.
+
+- [Web reference](https://developer.mozilla.org/en-US/docs/Web/CSS/text-shadow)
+
+#### `textTransform`
+
+Accepts one of the following values:
+
+- `` `none ``
+- `` `uppercase ``
+- `` `lowercase ``
+- `` `capitalize ``
+
+- [Web reference](https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform)
+
+#### `writingDirection`
+
+Accepts one of the following values:
+
+- `` `auto ``
+- `` `ltr ``
+- `` `rtl ``
+
+### Image Style Props
+
+#### `resizeMode`
+
+Accepts one of the following values:
+
+- `` `cover ``
+- `` `contain ``
+- `` `stretch ``
+- `` `repeat ``
+- `` `center ``
+
+Similar to CSS `background-size` values
+
+- [Web reference](https://developer.mozilla.org/en-US/docs/Web/CSS/background-size)
+
+#### `overlayColor`
+
+Accepts a `Color.t` (`string`).
+
+#### `tintColor`
+
+Accepts a `Color.t` (`string`).
+
+### Unsafe Style Props
+
+⚠️ _Use only as an escape hatch. Don't overuse this functions._
+
+In case you are using something unsupported by a specific platform, you can use
```

Comment:
reads as if the prop is not supported by the platform, perhaps better to write:
In case you are using an unsupported prop on another platform, you can use

Reply:
I changed to:
In case you want to something unsupported by this binding for a specific
platform, you can use `unsafeAddStyle` & `unsafeStyle`.